src/Providers/DataGrid/Wijmo/Features/FeatureBuilder.ts

```diff
@@ -207,7 +207,7 @@ namespace Providers.DataGrid.Wijmo.Feature {
         }
 
         private _makeToolTip(): FeatureBuilder {
-            this._makeItem(ToolTip);
+            this._features.toolTip = this._makeItem(ToolTip);
```

Comment:
Remove the upper case in the `T` of the tooltip.

Reply:
Done!